readme.md

```diff
@@ -83,6 +83,7 @@ This section contains libraries that are well-made and useful, but have not nece
 - [ALX](https://github.com/google-research/google-research/tree/master/alx) - Open-source library for distributed matrix factorization using Alternating Least Squares, more info in [_ALX: Large Scale Matrix Factorization on TPUs_](https://arxiv.org/abs/2112.02194).
 - [Diffrax](https://github.com/patrick-kidger/diffrax) - Numerical differential equation solvers in JAX. <img src="https://img.shields.io/github/stars/patrick-kidger/diffrax?style=social" align="center">
 - [tinygp](https://github.com/dfm/tinygp) - The _tiniest_ of Gaussian process libraries in JAX. <img src="https://img.shields.io/github/stars/dfm/tinygp?style=social" align="center">
+- [galax](https://github.com/yuanqing-wang/galax) - Graph-centric, flexible graph learning package.
```

Comment:
Please add the badge via `<img src="https://img.shields.io/github/stars/yuanqing-wang/galax?style=social" align="center">`